### PR TITLE
SwitchToWindow command implementation

### PR DESCRIPTION
**Pull request by zebraxxl**

### CODE REVIEW

**Comment by zebraxxl** [comment]
Requires https://github.com/2gis/Winium.Cruciatus/pull/51

**Comment by skyline-gleb** [comment]
@zebraxxl up Winium.Cruciatus version please

File: src/Winium.Desktop.Driver/CommandExecutors/SwitchToWindowExecutor.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    internal class SwitchToWindowExecutor : CommandExecutorBase
+    {
+        #region Methods
+
+        protected override string DoImpl()
+        {
+            var windowHandle = int.Parse(this.ExecutedCommand.Parameters["name"].ToString());
+
+            var handleProperty = AutomationElement.NativeWindowHandleProperty;
+            var window = CruciatusFactory.Root.FindElement(By.AutomationProperty(handleProperty, windowHandle));
```
Comment (skyline-gleb) [inline src/Winium.Desktop.Driver/CommandExecutors/SwitchToWindowExecutor.cs]: sorry, i missed what not have error handling when window not found:
``` c#
if (window == null)
{
    throw new AutomationException("Window cannot be found", ResponseStatus.NoSuchElement)
}
```

**Comment by skyline-gleb** [comment]
And update `Winium.Cruciatus` package version again (to v2.9.1), please, it will be very cool

File: src/Winium.Desktop.Driver/CommandExecutors/SwitchToWindowExecutor.cs (hunk 1 above)
Comment (zebraxxl) [inline src/Winium.Desktop.Driver/CommandExecutors/SwitchToWindowExecutor.cs]: Sorry, my fault.

**Comment by skyline-gleb** [comment]
:+1:

**Pull request merged**